esx_service/cli/vmdkops_admin.py

```diff
@@ -499,27 +499,27 @@ def policy_ls(args):
             active = 'Unused'
         rows.append([name, content.strip(), active])
 
-    print cli_table.create(header, rows)
+    print(cli_table.create(header, rows))
 
 
 def policy_update(args):
     output = vsan_policy.update(args.name, args.content)
     if output:
-        print output
+        print(output)
     else:
-        print 'Successfully updated policy: {0}'.format(args.name)
+        print('Successfully updated policy: {0}'.format(args.name))
 
 
 def status(args):
-    print "Version: {0}".format(get_version())
+    print("Version: {0}".format(get_version()))
     (status, pid) = get_service_status()
-    print "Status: {0}".format(status)
+    print("Status: {0}".format(status))
     if pid:
-        print "Pid: {0}".format(pid)
-        print "Port: {0}".format(get_listening_port(pid))
-    print "LogConfigFile: {0}".format(log_config.LOG_CONFIG_FILE)
-    print "LogFile: {0}".format(log_config.LOG_FILE)
-    print "LogLevel: {0}".format(log_config.get_log_level())
+        print("Pid: {0}".format(pid))
+        print("Port: {0}".format(get_listening_port(pid)))
+    print("LogConfigFile: {0}".format(log_config.LOG_CONFIG_FILE))
+    printr("LogFile: {0}".format(log_config.LOG_FILE))
```

Comment:
I just debugged the make all failure and tracked it to this line. Should be `print` instead of `printr`.